share/revng/test/tests/model/import/types/types.c.model.yml

```diff
@@ -0,0 +1,207 @@
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+---
+Segments:
+  - StartAddress:    "0x400000:Generic64"
```

Comment:
It seems you commited the model file as is, you need to clean it up from all the irrelevant things from the purpose of your test. In particular addresses, superfluous primitive types, size you're not interested in...

Reply:
I thought it was good to keep at least the key, `StartAddress` and `VirtualSize`; but OK, removed all of them and kept only the relevant types.